allensdk/brain_observatory/behavior/session_apis/data_io/__init__.py

```diff
@@ -1,8 +1,18 @@
+# raw data_io class for behavior
```

Comment:
I think it is specc'ed out in [Document data models for Behavior Ophys in AllenSDK](#1757) already:
it would be nice to have a doc for the relationships between all these classes (the ones called out in this `__init__.py`.
What do you think about generating some diagram as part of that, for instance, with [pyan3](https://github.com/Technologicat/pyan)?